source/fundamentals/connection/tls.txt

```diff
@@ -77,6 +77,26 @@ in the following ways:
    see the :manual:`SRV Connection Format </reference/connection-string/#srv-connection-format>`
    section in the Server manual.
 
+.. note:: Workaround for an "unsafe legacy renegotiation disabled" error
```

Comment:
I: [caps](https://www.mongodb.com/docs/meta/style-guide/style/titles-and-headings/#capitalization)

```suggestion
.. note:: Workaround for an "unsafe legacy renegotiation disabled" Error
```